README.md

```diff
@@ -118,11 +118,13 @@ Launch the application with the following commands:
     roslaunch cloudwatch_simulation [command] follow_route:=false dynamic_route:=true
     ``` 
 
+For navigation, you can generate a map with map generation plugin. See [this](#generate-occupancy-map-via-map-generation-plugin) for instructions.
+
 ![CloudWatchMetrics01.png](docs/images/BookstoreRVizPlan01.png)
 
 ### Run with a AWS Robomaker WorldForge world
 
-Pre-requisite: Generate a map for your worldforge exported world following these [instructions](#generate-map-for-your-world).
+Pre-requisite: Generate a map for your worldforge exported world following these [instructions](#generate-map-for-a-worldforge-world-with-default-config).
 
 Move the generated map file to cloudwatch_simulation package,
```

Comment:
This might be redundant and/or confusing if the genmap_script is used for map generation.. maybe could add a note clarifying this